### DISCUSSION TITLE
Medusa JS (specifically v2 when released)

### DISCUSSION THREAD

**Discussion by hybes** [coollabsio/coolify#3335] [Service Template Requests]
https://github.com/medusajs/medusa/

**Comment by hybes** [comment]
2.0 RC is now out and is likely near ready for deployment - I'm sure this can be looked at shortly.

**Reply by pavelstianko** [reply]
Medusa 2 is amazing but hard to deploy, so this would be awesome!

**Reply by alaistair76** [reply]
About to embark on a Medusa 2.0 project.. be great if there was a coolify template

**Comment by jnxmax** [comment]
Yes please! Add a Coolify template it would be amazing

**Comment by sianami123** [comment]
i need medusa template on coolify too

**Comment by nexthis** [comment]
instruction:
https://github.com/nexthis/medusa-coolify

**Reply by yipfram** [reply]
This might be a stupid question but why create a new repository and not the compose file directly as a template? 
I didn't saw any "build" part or reference to the repo in the docker compose file... 

I am new to this, happy to learn :)
